go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -1742,6 +1743,12 @@ func FnMonthName(yield Query) {
 	}
 }
 
+func FnLastDay(yield Query) {
+	for _, d := range inputConversions {
+		yield(fmt.Sprintf("LAST_DAY(%s)", d), nil)
+	}
```

Comment:
Next to these inputs, we should also test it against a few more inputs explicitly to test edge case behaviors here. Like last of December, January & February years (with / without leap years etc).

Reply:
done.